PendingReleaseNotes

```diff
@@ -486,3 +486,58 @@
   (``auth_client_required=cephx`` and ``ms_mon_client_mode=secure``).
   If you have cephx authentication disabled on your cluster, you may
   need to adjust these settings for RGW.
+>=18.0.0
```

Comment:
Do we need 18.0.0 stuff in quincy's `PendingReleaseNotes`?

Reply:
Thanks Ilya, I see that the commit for PendingReleaseNotes did not apply cleanly. I'll have to rebase it.